storage/innobase/handler/ha_innodb.cc

```diff
@@ -24265,8 +24266,8 @@ static MYSQL_SYSVAR_UINT(
     encryption_threads, srv_n_fil_crypt_threads_requested, PLUGIN_VAR_RQCMDARG,
     "Number of threads performing background key rotation and "
     "scrubbing",
-    innodb_encryption_threads_validate, innodb_encryption_threads_update,
-    srv_n_fil_crypt_threads_requested, 0, UINT_MAX32, 0);
+    innodb_encryption_threads_validate, innodb_encryption_threads_update, 0, 0,
```

Comment:
why did you remove srv_n_fil_crypt_threads_requested?

Reply:
Hmm, the variable is assigned to 0, so the default is 0. OK to use explicit value.